torch/quantization/fx/quantization_patterns.py

```diff
@@ -109,9 +110,12 @@ def __init__(self, quantizer: QuantizerCls, node: Node):
             torch.mul: torch.ops.quantized.mul_relu,
         }
         # corresponding quantized op
-        self.qop = qbin_relu_op_mapping[self.bop] \
-            if self.relu_node is not None \
-            else qbin_op_mapping[self.bop]  # type: ignore
+        if self.bop in qbin_op_mapping:
```

Comment:
optional nit (probably for an earlier PR): `bop` is hard to think about, maybe `binary_op`?  same for `qop`

Reply:
sure